files/config_validate.rb

```diff
@@ -0,0 +1,76 @@
+#!/opt/puppetlabs/puppet/bin/ruby
```

Comment:
now this is tricky. This assumes an installtion with the Puppet AIO packages. On Arch Linux, FreeBSD and Gentoo, the ruby binary is at a different location. Maybe `#!/usr/bin/env ruby` works.

Reply:
But the OS might have no ruby installed and `/opt/puppetlabs/puppet/bin` is not in $PATH usually.. not sure how to deal with this.

Reply:
I think, but I'm not sure, that puppet adds `/opt/puppetlabs/puppet/bin` to `$PATH` during agent runs.

Reply:
the parameter is now customizable and the interpreter is set to `#!/usr/bin/env ruby`